pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -655,6 +659,9 @@ private void addNewRow(int docId, GenericRow row) {
             case DOUBLE:
               forwardIndex.setDouble(docId, (Double) value);
               break;
+            case BIG_DECIMAL:
```

Comment:
Do we have tests for mutable forward index with big decimal ? Or may be a test in `RealtimeClusterIntegrationTest` that ingests and queries data from a consuming segment ?

Reply:
Based on Jackie's input, removed the test. A separate PR will contain more thorough tests for BIG_DECIMAL feature.